### DISCUSSION TITLE
blobwifi not working

### DISCUSSION THREAD

**Discussion by Perplexedykwhat** [3kh0/ext-remover#1305] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
help blobwifi is not working, whenever I try to export the .onc file with the selected wifi, it gives me an error.

**Comment by ghost** [comment, marked as answer]
If you are using the ext-remover.net or chrose.pages.dev version, go to the official site because that worked for me. But this is patched v128 and above according to TitaniumNetwork's kajigs pages.

**Reply by Perplexedykwhat** [reply]
> If you are using the ext-remover.net or chrose.pages.dev version, go to the official site because that worked for me. But this is patched v128 and above according to TitaniumNetwork's kajigs pages.  

the official website is blocked by the internet 😭😭😭
